File: bulk/java/LogInjection01.java
// CWE-117: Log Injection
import org.slf4j.*;
public class LogInjection01 {
    private static final Logger log = LoggerFactory.getLogger(LogInjection01.class);
    public void logLogin(String username) {
        log.info("Login attempt: " + username); // CRLF in username injects fake entries
    }
}

File: bulk/csharp/LogInjection01.cs
// CWE-117: Log Injection
using Microsoft.Extensions.Logging;
public class LogInjection01 {
    private readonly ILogger _logger;
    public LogInjection01(ILogger<LogInjection01> logger) => _logger = logger;
    public void LogLogin(string username) {
        _logger.LogInformation("Login: " + username); // CRLF injection
    }
}
